src/selectors/__tests__/expandThoughts.ts

```diff
@@ -282,11 +283,27 @@ describe('=pin', () => {
     const stateNew = reducerFlow(steps)(initialState())
 
     expect(isContextExpanded(stateNew, ['a'])).toBeTruthy()
-    expect(isContextExpanded(stateNew, ['a', 'b'])).toBeFalsy()
```

Comment:
This test is actually correct as it was. `=pin/false` takes precedence over `=pinChildren/true`.

`=pin/false` is only ignored when the cursor is on the thought, e.g. on `a/b`.